doc/dynamo0p3.rst

```diff
@@ -1086,18 +1101,32 @@ copy_scaled_field
 
 Multiplies a field by a scalar and stores the result in a second field: ::
   
-  field2(:) = value * field1(:)
+   field2(:) = value * field1(:)
 
 where:
 
 * real(r_def), intent(in) :: *value*
 * type(field_type), intent(in) :: *field1*
 * type(field_type), intent(out) :: *field2*
 
-divide_field
-++++++++++++
+divide_fields
++++++++++++++
+
+**divide_fields** (*field1*, *field2*, *field3*)
 
-**divide_field** (*field1*, *field2*)
+Divides the first field by the second and returns the result in the third: ::
+
+   field3(:) = field1(:) / field2(:)
+
+where:
+
+* type(field_type), intent(in) :: *field1*, *field2*
+* type(field_type), intent(out) :: *field3*
+
+inc_divide_field
```

Comment:
Added the explanation. I also moved "inc_multiply_field" below "multiply_fields" to be consistent with the "divide" example.